requirements.in.txt

```diff
@@ -88,3 +88,4 @@ itsdangerous==2.0.1
 MarkupSafe==2.0.1
 certifi>=2022.12.7
 setuptools>=65.5.1
+cryptography>=39.0.1
```

Comment:
```suggestion
```

Reply:
It is already pinned 39.0.1 in the txt file, making is not necessary in the in.txt file any more.